pyart/util/radar_utils.py

```diff
@@ -104,6 +104,203 @@ def to_vpt(radar, single_scan=True):
     return
 
 
+def cut_radar(radar, field_names, rng_min=None, rng_max=None, ele_min=None,
```

Comment:
This all looks great! One small suggestion - what would you think about changing the name here to subset_radar?